editor.planx.uk/src/@planx/components/Calculate/Editor.tsx

```diff
@@ -72,7 +72,16 @@ export default function Component(props: Props) {
     <form onSubmit={formik.handleSubmit} id="modal">
       <ModalSection>
         <ModalSectionContent title="Calculate" Icon={ICONS[TYPES.Calculate]}>
-          <Typography variant="body2">
+          <InputRow>
+            <Input
+              format="large"
+              placeholder="Title"
+              name="title"
+              value={formik.values.title}
+              onChange={formik.handleChange}
+            />
+          </InputRow>
+          <Typography variant="body2" sx={{mt: 2}}>
```

Comment:
nit: I think the order of Typography block _then_ title input would look a bit less choppy here!

Reply:
Good shout, swapped in [e406e3c](https://github.com/theopensystemslab/planx-new/pull/2873/commits/e406e3cd3ff92e99563132dcadc276b4cb893bd5)

Screenshot in PR updated also ✅